nservicebus/msmq/dead-letter-queues.md

```diff
@@ -15,3 +15,23 @@ NOTE: If DLQ is enabled messages will remain in the senders outgoing queue until
 
 
 partial: config
+
+
+### Monitoring DLQ
+
+Misconfiguration of [routing](/nservicebus/messaging/routing) may result in messages ending up in the DLQ. It is very important to monitor the DLQ in order to detect potential routing configuration errors or other situations that may lead to messages being moved to the dead letter queue.
```

Comment:
There are more reasons like, purging queues, sending tx message to non tx queues etc. Should we mention that?

Perhaps also add a link to https://msdn.microsoft.com/en-us/library/ms706227(v=vs.85).aspx